salt/cloud/clouds/qingcloud.py

```diff
@@ -0,0 +1,845 @@
+# -*- coding: utf-8 -*-
+'''
+QingCloud Cloud Module
+======================
+
+.. versionadded:: 2015.2
```

Comment:
This should actually be a `.. versionadded:: Beryllium` since this is a new feature going into develop.